airbyte-integrations/connectors/source-avni/source_avni/schemas/subjects.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Make sure all schemas follow the https://docs.airbyte.com/connector-development/schema-reference/